corehq/apps/reports/tasks.py

```diff
@@ -352,44 +356,50 @@ def find_question_id(form, value):
 
         return None
 
-    case_ids = set()
 
-    def extract_form_info(form, properties=None, case_ids=case_ids):
-        unknown_number = 0
-        # get case ids
-        case_blocks = extract_case_blocks(form.form_data)
-        cases = {c['@case_id'] for c in case_blocks}
-        case_ids |= cases
-
-        form_info = {
-            'form': form,
-            'attachments': list(),
-            'name': form.name or "unknown form",
-            'user': form.user_id or "unknown_user",
-            'cases': cases,
-            'id': form.form_id,
-        }
-        for attachment_name, attachment in form.attachments.iteritems():
-            content_type = getattr(attachment, 'content_type', attachment['content_type'])
-            if content_type == 'text/xml':
-                continue
-            try:
-                question_id = unicode(u'-'.join(find_question_id(form.form_data, attachment_name)))
-            except TypeError:
-                question_id = unicode(u'unknown' + unicode(unknown_number))
-                unknown_number += 1
-
-            if not properties or question_id in properties:
-                extension = unicode(os.path.splitext(attachment_name)[1])
-                form_info['attachments'].append({
-                    'size': getattr(attachment, 'length', attachment['length']),
-                    'name': attachment_name,
-                    'question_id': question_id,
-                    'extension': extension,
-                    'timestamp': form.received_on.timetuple(),
-                })
-
-        return form_info
+    unknown_number = 0
+    # get case ids
+    case_blocks = extract_case_blocks(form.form_data)
+    cases = {c['@case_id'] for c in case_blocks}
+    case_ids |= cases
+
+    form_info = {
+        'form': form,
+        'attachments': list(),
+        'name': form.name or "unknown form",
+        'user': form.user_id or "unknown_user",
+        'cases': cases,
+        'id': form.form_id,
+    }
+    for attachment_name, attachment in form.attachments.iteritems():
+        content_type = getattr(attachment, 'content_type',
+                               attachment['content_type'])
+        if content_type == 'text/xml':
+            continue
+        try:
+            question_id = unicode(
+                u'-'.join(find_question_id(form.form_data, attachment_name)))
+        except TypeError:
+            question_id = unicode(u'unknown' + unicode(unknown_number))
```

Comment:
nit: think this could probably just be `u'unknown' + unicode(unknown_number)` realize you didn't write that though